perception/autoware_lidar_centerpoint/include/autoware/lidar_centerpoint/centerpoint_config.hpp

```diff
@@ -116,7 +116,7 @@ class CenterPointConfig
   std::size_t head_out_rot_size_{2};
   std::size_t head_out_vel_size_{2};
 
-  // post-process params
+  // post-processing params
```

Comment:
Is this change necessary?
```suggestion
  // post-process params
```

Reply:
Other than grammar, no

Reply:
This is just a nits, but can we revert this change?